.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,12 +2,15 @@
 
 ## PR Checklist
 
-- [ ] Has Pytest style unit tests
-- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
-- [ ] New features are documented, with examples if plot related
-- [ ] Documentation is sphinx and numpydoc compliant
-- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
-- [ ] Documented in doc/api/next_api_changes/* if API changed in a backward-incompatible way
+<!-- Please mark any checkboxes that do not apply to this PR as [N/A]. -->
+
+- [ ] Has pytest style unit tests (and `pytest` passes).
+- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
+- [ ] New features are documented, with examples if plot related.
+- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
+- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and `pydocstyle<4` and run `flake8 --docstring-convention=all`).
+- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
```

Comment:
don't need the `there`

Reply:
I think without it it's ambiguous where to find the README.

Also, this language was already there.

Reply:
Ugh, it's an ambiguous reference & yes can kick that can down to a different PR.

Reply:
Thanks!